code/obj/artifacts/artifact_items/talisman.dm

```diff
@@ -0,0 +1,138 @@
+#define SWIFTNESS "swiftness"
+#define FORTUNE 2
+#define SPACEFARING 3
+#define ELEMENTS 4
+#define STRENGTH 5
+#define PROTECTION 6
+
+/*
+-Random cold protection (from space) from 25% - 75% - the artifact feels somewhat cold
+-Random set of heat + cold protection from 0% - 25% each - feels hot and cold in different spots
+-Random incoming brute, burn, tox reduction from 0 - 10% each - holding it makes you feel safe somehow
+-Random max health increase from 0 - 50 - you get a sense of power from it
+-Random amounts of money appearing in your inventory between 0 - 200 credits. does not add it if your inventory is full. - holding it makes you feel lucky somehow
+-Random movement speed increase from 0 - 5% - you feel drafts of air surrounding this artifact/you feel air moving quickly around this artifact
+*/
+
+/obj/item/artifact/talisman
```

Comment:
Any chance you could make this an `/obj/item/clothing/suit`? It would be neat to wear them sort of like an amulet around your neck. Kind of how the HoS medal works. Still has the same effects, but would allow some interesting extra slot management.

Reply:
Could, but it would require sprites and all + removes the part of this in that it provides its magical effects without worn, if restricted to clothing

Reply:
Wouldn't *need* sprites technically, and it should be easy enough to make some variants using the HoS medal as reference for size and just making 1 for each origin.

The magical effect should remain in pocket or worn is what I'm saying. I'm not arguing to change that. But it allows you to balance having it in your pocket or suit slot depending on convenience.

Reply:
I probably won't plan on it for now, agreed it would be cool, but I think wearing it is more on the extra side